### ISSUE TITLE
[6] 14: Add blog subscription module and form

### ISSUE THREAD

**Issue by amberreeves** [InteractiveMechanics/CollegeWomen_Drupal7#51]
- [ ] Add the Drupal 7 Mailchimp module 
- [ ] Add a form to the blog for users to sign up

@mtedeschi to provide design

**Comment by mtedeschi** [comment]
Start with Drupal 7 Mailchimp module.

**Comment by mtedeschi** [comment]
@amberreeves can you request their Mailchimp information?

**Comment by amberreeves** [comment]
i asked evan for this last week. still waiting on a response.

**Comment by amberreeves** [comment]
@cworthen see design and annotations

https://marvelapp.com/2h8b1hj/screen/37146139

still waiting for credentials

**Comment by amberreeves** [comment]
Mailchimp credentials
user: CollegeWomenHistory@gmail.com 
password: M@ryLy0nRulz

**Comment by cworthen** [comment]
@amberreeves  I tried to login into mail chimp with creds provided and it couldn't find the account, should I create an account with the creds provided ?

**Comment by amberreeves** [comment]
i'm checking with the client now. i'll let you know as soon as i have more info!

**Comment by amberreeves** [comment]
ok she said the user name is just 

 CollegeWomenHistory

(no email address attached)
and sorry for the hold up!

**Comment by cworthen** [comment]
Thanks @amberreeves!

**Comment by cworthen** [comment]
Side note, I'll have to a tiny bit of styling to the form for mobile screens.

**Comment by amberreeves** [comment]
@mtedeschi do you want to give direction on mobile styling?

**Comment by mtedeschi** [comment]
A few adjustments (make sure to reference the design from Marvel: https://marvelapp.com/2h8b1hj/screen/37146136).

- [ ] The block should only appear on blog post nodes, not every page.
- [ ] Revise styling to match design treatments (apologies, I realize I left in placeholder text when I provided you the designs: It could just be "Sign up for updates", the input, and "Submit ->")
- [ ] Is it functioning, currently? Confirm that when you submit an email address it does direct you to the Mailchimp signup form.

**Comment by cworthen** [comment]
@mike, confirmation that form is working, I used test@interactive: 
![screen shot 2018-02-21 at 10 54 46 pm](https://user-images.githubusercontent.com/16109830/36519600-76a103e8-175a-11e8-88ca-88af3540f24c.png)

**Comment by mtedeschi** [comment]
@cworthen: sorry, I think I understand. So when you submit the form, it adds the user to the Mailchimp list? If that's what happens, that's great. I just assumed it would behave like it does on our site (see here for an example: http://interactivemechanics.com/news/).

**Comment by cworthen** [comment]
@mike, exactly, when a user signs up, they're automatically added to the list

**Comment by mtedeschi** [comment]
Made a few minor styling adjustments. Good to go on dev.
